src/components/App/App.css

```diff
@@ -1,8 +1,9 @@
 .App {
   background-color: #d0c5b9;
-  max-height: 100vh;
+  min-height: 100vh;
   display: flex;
   flex-direction: column;
-  font-family: 'Satisfy', cursive;
+  font-family: "Satisfy", cursive;
   justify-content: space-between;
+  padding-bottom: 5rem;
```

Comment:
Making a note of this rule for min-height vs max-height and padding bottom bc it's working effectively on my branch with max height and no padding.